vue/components/ui/molecules/filter/filter.vue

```diff
@@ -334,6 +363,30 @@ export const Filter = {
                 return false;
             }
 
+            if (auto) {
+                const oldItems = this.items.map(item => item.id);
+                const newItems = items.map(item => item.id);
+                const freshItems = newItems.reduce((a = 0, b) => a + !oldItems.includes(b), 0);
+
+                // checks if there's any new item and appends to the list
+                if (freshItems) {
+                    options.start = this.items.length - 1;
+                    options.limit = freshItems;
+                    const refreshedItems = await this.getItems(options);
+                    this.tableTransition = "fade";
+                    this.items = [...items, ...refreshedItems];
+                    this.itemsToLoad = true;
+

```

Comment:
Add  `this.$emit("auto:refresh:stop");` here?

You have that before returning a few lines below